docs/index.md

```diff
@@ -23,13 +40,97 @@ Additional ways to get in contact with the team:
 ## Code
 
 The current production deployment of Push is a Python-based websocket server
-named **autopush**, developed on Github.
+named **autopush**, developed on Github. The Push Service team has several
+repositories for development of the Push Server, and supporting tooling.
+
+- [autopush](https://github.com/mozilla-services/autopush) - Actively developed
+  Push server
+- [pushgo](https://github.com/mozilla-services/pushgo) - Legacy Go-based Push
+  server (no longer developed or deployed)
+- [push-tester](https://github.com/bbangert/push-tester) - SimplePush load
+  tester (no longer developed)
+
+## Terminology
+
+The Push Service uses a range of terminology that can be difficult at first to
+keep straight. This is a list of commonly used terms and phrases with their
+meaning in the context of Push.
+
+For an overview of how these terms fit together to deliver a push message to a
+user-agent, [see the Firefox Push Notification high-level
+diagram](https://wiki.mozilla.org/Firefox/Push_Notifications#Technologies). This
+page also includes descriptions of related technologies that help to deliver a
+cohesive Push Notification experience.
+
+Where applicable, these match the [Webpush Spec Terminology][wpst].
+
+Application
+: Both the sender and ultimate consumer of push messages. Many applications
+  have components that are run on a user agent and other components that run on
+  servers.
+
+Application Server
+: The component of an application that runs on a server and requests the
+  delivery of a push message.
+
+Channel ID
+: Legacy terminology for a Push Message Subscription utilized by SimplePush and
+  the websocket client protocol.
+
+Endpoint
+: A REST-ful HTTP URL uniquely associated with a Push Message Subscription. This
+  is referred to as a "push resource" in the [WebPush specification][wp].
+
+Push Message Subscription
+: A message delivery context that is established between the user agent and the
+  push service and shared with the application server.  All push messages are
+  associated with a push message subscription.
+
+Push Message
+: A message sent from an application server to a user agent via a push service.
+
+Push Message Receipt
+: A message delivery confirmation sent from the push service to the application
+  server.
+
+Push Service
+: A service that delivers push messages to user agents.
+
+SimplePush
+: Firefox OS specific Push system that carries no data, only an incrementing
+  version number. The Mozilla Push Service continues to support this legacy API
+  for Firefox OS devices.
+
+UAID
+: A globally unique UserAgent ID. Used by the Push Service to associate clients
+  utilizing the websocket protocol with their associated channel ID's.
```

Comment:
Nit: Should we note that this is a UUID4 here or suppose that the developer will later read that?